src/ucp/wireup/wireup_cm.c

```diff
@@ -478,6 +475,9 @@ static unsigned ucp_cm_client_connect_progress(void *arg)
     }
 
     if (!context->config.ext.cm_use_all_devices) {
+        /* Restore initial configuration from tmp_ep created for packing local
```

Comment:
start the comment using lowercase and remove trailing `.` as in another place where the same comment is added